schema/spec.md

```diff
@@ -150,13 +154,20 @@ being created.
 In "semantic Versioning" terms, you can think of a **schema** as a "major
 version" and the **schema Versions** as "minor versions".
 
+## Schema Registry Model
+
+The formal xRegistry extension model of the Schema Registry resides in the
+[model.json](model.json) file.
```

Comment:
Just wondering, why is the above line only in the schema model and not the others?

Reply:
And should these labels be in those?

Also, I just realized, we should probably add `?` to the end of each label (in all docs) since w/o that people will think they're required labels.

Reply:
yes, they should have ` ?` suffix. 

> Just wondering, why is the above line only in the schema model and not the others?

I thought It was already in the other files and missing here, so I added it for consistency.

Reply:
Oh interesting, I never really noticed it before. Thanks